### PR TITLE
add operation docs

### PR DESCRIPTION
**Pull request by zhouyun-zoe**
1. rewrite getting_started
2. create `operation` section to hold 
   - install
   - configuration
   - protocol upgrade

It is necessary to add more docs to fill the `operation` section, refer to #78.
Need you guys help @jiangxianliang007 @kaikai1024

### CODE REVIEW

File: docs/cita/getting-started.md [hunk 1]
```diff
@@ -6,201 +6,195 @@ title: 快速入门
 CITA 是一个开源的区块链内核，任何人都可以基于 CITA 来搭建属于自己的一条区块链，在本文档中我们将带你搭建一条简单的链并运行其中的节点。
 
 > * 如果你想一键搭建属于你自己的链，你可以选择租用 CITA 的云服务。只需根据您的需求，在云服务平台选择适合自己的方案直接租用，帮你省去准备服务器以及部署 CITA 的一系列操作。具体请参考[云服务支持](./huawei)。
-> * 如果你想在 CITA 上直接开发您的应用，我们建议你使用我们已经搭好的 CITA 测试链。测试链的水龙头的地址为：https://dapp.cryptape.com/faucet/, 可以在这里领取 Testnet 的代币。测试链的内核 CITA 的版本为 [`v0.23.1`](https://github.com/cryptape/cita/releases/tag/v0.23.1)。该测试链由4个节点组成，各节点的 ip 地址和端口如下：
+> * 如果你想在 CITA 上直接开发您的应用，我们建议你使用我们已经搭好的 CITA 测试链。
```
Comment (kaikai1024) [inline docs/cita/getting-started.md]: Why delete the info about test-chain?

File: docs/cita/getting-started.md [hunk 2]
```diff
@@ -6,201 +6,195 @@ title: 快速入门
...
-```
-$ sudo docker run hello-world
-```
+### 安装 CITA 客户端工具
 
-### 获取 Docker 镜像
+1. 创建目录
 
-CITA 的 Docker 镜像托管在 [DockerHub](https://hub.docker.com/r/cita/cita-build/)。
+   ```
```
Comment (kaikai1024) [inline docs/cita/getting-started.md:35]: ~~It seems that the indentation will cause a problem.~~

~~And all the code block.~~

Forget this.

File: docs/cita/getting-started.md [hunk 3]
```diff
@@ -6,201 +6,195 @@ title: 快速入门
...
-可以使用 `docker pull` 命令直接从 DockerHub 获取， 参见 [在线资料](https://yeasy.gitbooks.io/docker_practice/content/image/pull.html)。
+2. 切换目录
 
-对于内网环境，也可以通过 `docker save` 和 `docker load` 传递镜像， 参见 [在线资料](https://yeasy.gitbooks.io/docker_practice/content/image/other.html)。
+   ```
+   $ cd /data/cita/
+   ```
 
-## 编译 CITA
+3. 下载CITA-CLI安装包
```
Comment (kaikai1024) [inline docs/cita/getting-started.md]:
```suggestion
3. 下载 CITA-CLI 安装包
```

File: docs/cita/getting-started.md [hunk 4]
```diff
@@ -6,201 +6,195 @@ title: 快速入门
...
+   "address": "0x141d051b1b1922bf686f5df8aad45cefbcb0b696",
+   "private": "0xa2c4ce22f3ee18d4ca6e560d4a5a6b54823773348b95cd95afb3ee2843ff7768",
+   "public": "0x37aa832ae77a580c2c305dfee234af424a90274b236b8925d7c452e1834f3fa34cd87178c2116d34dc816d0b06c869b01409a328d7a28fcfcfca994f6095d166"
+   }
+   ```
 
-```shell
-$ ./env.sh make release
-```
+   > 注："address": "0x141d051b1b1922bf686f5df8aad45cefbcb0b696" 为超级管理员帐号地址，下面的节点管理操作中会频繁使用
```
Comment (kaikai1024) [inline docs/cita/getting-started.md]: Duplate L-69

File: docs/cita/getting-started.md [hunk 5]
```diff
@@ -6,201 +6,195 @@ title: 快速入门
...
->   ```
-
-## 部署CITA
-
-### 配置节点
+2. 下载CITA安装包
 
-* 先切换到发布件目录
+   ```
+   $ wget https://github.com/cryptape/cita/releases/download/v0.20.2/cita_secp256k1_sha3.tar.gz
```
Comment (kaikai1024) [inline docs/cita/getting-started.md]: Use the latest version

File: docs/cita/getting-started.md [hunk 6]
```diff
@@ -6,201 +6,195 @@ title: 快速入门
...
+   ```
+   $ cd cita_secp256k1_sha3
+   ```
+5. 初始化链 （super_admin 地址是超级管理员账号即 cita-cli 生成，--nodes 是要部署的节点地址（IP:Port），RPC 端口从 1337 开始 递增；4个节点（1337、1338、1339、1340））
 
-    ```shell
-    $ cd cita_secp256k1_sha3/
-    ```
+   ```
+   $ ./env.sh ./scripts/create_cita_config.py create --super_admin "0x141d051b1b1922bf686f5df8aad45cefbcb0b696" --nodes "127.0.0.1:4000,127.0.0.1:4001,127.0.0.1:4002,127.0.0.1:4003"
```
Comment (kaikai1024) [inline docs/cita/getting-started.md]:
```suggestion
   $ bin/cita create --super_admin "0x141d051b1b1922bf686f5df8aad45cefbcb0b696" --nodes "127.0.0.1:4000,127.0.0.1:4001,127.0.0.1:4002,127.0.0.1:4003"
```

File: docs/cita/getting-started.md [hunk 7]
```diff
@@ -6,201 +6,195 @@ title: 快速入门
...
-> 下同
+### 启动 CITA
 
+1. 启动节点 0
 
-  ```shell
-  $ bin/cita create --super_admin "0x4b5ae4567ad5d9fb92bc9afd6a657e6fa13a2523" --nodes "127.0.0.1:4000,127.0.0.1:4001,127.0.0.1:4002,127.0.0.1:4003"
-  ```
+   ```
+   $ ./env.sh ./bin/cita setup test-chain/0
```
Comment (kaikai1024) [inline docs/cita/getting-started.md]:
```suggestion
   $ bin/cita setup test-chain/0
```

File: docs/cita/getting-started.md [hunk 8]
```diff
@@ -6,201 +6,195 @@ title: 快速入门
...
+   $ ./daemon.sh ./bin/cita start test-chain/0
```
Comment (kaikai1024) [inline docs/cita/getting-started.md]:
```suggestion
   $ bin/cita start test-chain/0
```

Comment (kaikai1024) [reply docs/cita/getting-started.md]: Same suggestion to node 1-3

File: docs/cita/getting-started.md [hunk 9]
```diff
@@ -6,201 +6,195 @@ title: 快速入门
...
-where <command> is one of the following:
-    { help | create | port | setup | start | stop | restart
-      ping | top | backup | clean | logs | logrotate }
-Run `cita help` for more detailed information.
-```
+1. 查看进程是否启动成功
 
-### 启动节点
+   ```
+   $ ps -ef | grep cita- |grep -v grep |wc -l && ps -ef | grep cita- |grep -v grep
```
Comment (kaikai1024) [inline docs/cita/getting-started.md]: `ps aux|grep cita`

File: docs/cita/getting-started.md [hunk 10]
```diff
@@ -6,201 +6,195 @@ title: 快速入门
...
 
-执行以下命令依次启动四个节点，该命令正常情况下不会返回，节点后台运行。
-
-```shell
-$ bin/cita start test-chain/0
-$ bin/cita start test-chain/1
-$ bin/cita start test-chain/2
-$ bin/cita start test-chain/3
-```
+   返回结果可以看到启动了28个进程（每个节点有7个服务*4个节点）
```
Comment (kaikai1024) [inline docs/cita/getting-started.md]: whitspace between number and word.
Same suggestion to the next part.

File: docs/cita/operation/install.md [hunk 11]
```diff
@@ -0,0 +1,262 @@
+---
+id: install
+title: 部署 CITA
+---
+
+CITA 是一个开源的区块链内核，任何人都可以基于 CITA 来搭建属于自己的一条区块链，在本文档中我们将带你搭建一条简单的链并运行其中的节点。
```
Comment (kaikai1024) [inline docs/cita/operation/install.md]: It's same with the `getting-start` part.

**Comment by kaikai1024** [comment]
Please declare it with a comment when you deal with the feedback.
Then reviewers can get notificitions.

**Comment by zhouyun-zoe** [comment]
@kaikai1024 done

**Comment by kaikai1024** [comment]
cool. would you please post some previews?

**Comment by zhouyun-zoe** [comment]
@kaikai1024 

屏幕快照 2019-05-10 上午10 40 17
屏幕快照 2019-05-10 上午10 40 57
屏幕快照 2019-05-10 上午10 37 32

**Review by kaikai1024 — APPROVED** [review]

**Pull request merged**